Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -17,6 +17,16 @@ public struct AssetViewModel: Identifiable, Equatable {
   }
 }
 
+public struct NFTAssetViewModel: Identifiable, Equatable {
```

Comment:
nit: Doesn't need to be public, isn't used outside of BraveWallet module

Reply:
Made internal in [d83aa86](https://github.com/brave/brave-ios/pull/6380/commits/d83aa86134c9d71543df6dbe78e741aff58e92af)